omnibus/config/software/datadog-agent-integrations.rb

```diff
@@ -167,11 +170,34 @@
       end
     end
 
+    if windows?
+      static_reqs_file = "#{windows_safe_path(install_dir)}\\#{agent_requirements_file}"
+    else
+      static_reqs_file = "#{install_dir}/#{agent_requirements_file}"
+
+    # Remove any blacklisted requirements from the static-environment req file
+    tmp = []
+    File.open("#{static_reqs_file}", 'r+') do |f|
+      f.each_line do |line|
+        tmp.push(line)
+      end
+    end
+
+    File.open("#{static_reqs_file}", 'w+') do |f|
```

Comment:
I'd use a different file to write the filtered reqs to make things more obvious

Reply:
I can do that, just have to make sure thats the file we use to `compile` with, which shouldn't be an issue.